exercises-latex/example-lab-report/loesung/v16516/content/auswertung.tex

```diff
@@ -0,0 +1,121 @@
+\section{Auswertung}
+\label{sec:Auswertung}
+
+
+Die für diesen Versuch relevanten physikalischen Größen für Kugel und Zylinder sind 
+
+\begin{align}
+  \label{eq:physical-properties}
+  m_\text{Z} &= \input{mass_ball.tex} & m_\text{K} &= \input{mass_ball.tex}\nonumber\\
+  r_{\text{Z},\text{i}} &= \input{../build/radius-inner_cylinder.tex} & r_\text{K} &= \input{radius_ball.tex}\\
+  r_{\text{Z},\text{a}} &= \input{../build/radius-outer_cylinder.tex} &&\nonumber
+\end{align}
+
+Berechnet wurde aus diesen Größen das jeweilige, theoretiche Trägheitsmoment
+
```

Comment:
I would not leave blank lines between equation and alignment environments, this creates weird spaces